core/arch/arm/tee/svc_cache.c

```diff
@@ -60,5 +59,5 @@ TEE_Result syscall_cache_operation(void *va, size_t len,
 	if (res != TEE_SUCCESS)
 		return TEE_ERROR_ACCESS_DENIED;
 
-	return cache_operation(op, va, len);
+	return cache_operation((enum utee_cache_operation)op, va, len);
```

Comment:
So what is the purpose of this cast? Does it have any technical advantage or is it purely cosmetic?